Datadog.Trace.sln

```diff
@@ -732,6 +734,7 @@ Global
 		{1A5E9F40-F3A5-4B59-9898-3DCD65C459C3}.Release|x64.Build.0 = Release|x64
 		{1A5E9F40-F3A5-4B59-9898-3DCD65C459C3}.Release|x86.ActiveCfg = Release|x86
 		{1A5E9F40-F3A5-4B59-9898-3DCD65C459C3}.Release|x86.Build.0 = Release|x86
+<<<<<<< HEAD
```

Comment:
Merge conflict?

Reply:
Yeah, that's what I get for manually resolving a merge conflict.